### PR TITLE
torch.nn.PixelShuffle crash with floating point exception when input has 0 size in the last three dimensions 

### PR DESCRIPTION
**Pull request by lucyjin3**
Fixes Issue #85155

### CODE REVIEW

**Comment by lucyjin3** [comment]
Just signed the Facebook CLA check.

**Comment by lucyjin3** [comment]
@mruberry Are you allowed to review my work? I need approval :(

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Build
```
Comment (mruberry) [inline .github/workflows/build.yml:1]: What's all this?

**Comment by mruberry** [comment]
This PR includes a new workflow which is not related to PixelShuffle.

**Pull request closed without merging**